ci/test/runtime-conf

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -ex
+# shellcheck source=./ci/test/common
+source ./ci/test/common
+init_test_repo
+cd "$test_repo_path"
+
+cat <<EOF > settings.py
+runtime_configuration = {
```

Comment:
we can name this `engine_configuration` instead. Also, we can remove the `docker` entry as that allows us to make the mix of kwargs (in the `mix_with_runtime_config` function) agnostic to each engine